### PR TITLE
chore: added indicio mainnet and demonet to ledgers

### PR DESCRIPTION
**Pull request by sabejensen**
Signed-off-by: seajensen <seajensen@gmail.com>

# Summary of Changes

Added Indicio main net and demo net to ledgers

# Related Issues

https://indicio.atlassian.net/browse/AMO-405

# Pull Request Checklist

Tick all boxes below to demonstrate that you have completed the respective task. If the item does not apply to your this PR **check it anyway** to make it apparent that there's nothing to do.

- [x] All commits contain a DCO `Signed-off-by` line (we use the [DCO GitHub app](https://github.com/apps/dco) to enforce this);
- [x] Updated LICENSE-3RD-PARTY.md for any added dependencies or vendored components;
- [x] Updated documentation as needed for changed code and new or modified features;
- [x] Added sufficient [tests](../__tests__/) so that overall code coverage is not reduced.

If you have _any_ questions to _any_ of the points above, just **submit and ask**! This checklist is here to _help_ you, not to deter you from contributing!

Pro Tip 🤓

- Read our [contribution guide](../CONTRIBUTING.md) at least once; it will save you a few review cycles!
- Your PR will likely not be reviewed until all the above boxes are checked and all automated tests have passed.

_PR template adapted from the Python attrs project._

### CODE REVIEW

**Review by jleach — CHANGES_REQUESTED** [review]
Can be merged once the lint issues are fixed.

**Comment by jleach** [comment]
@seajensen Able to pull in the changes from the base branch so we can get this change merged in. Ty.

**Comment by sabejensen** [comment]
> @seajensen Able to pull in the changes from the base branch so we can get this change merged in. Ty.

Should be fixed now

**Review by jleach — DISMISSED** [review]

File: core/configs/ledgers/indy/indicio-main-net/genesis-file.ts [hunk 1]
```diff
@@ -0,0 +1,4 @@
...
+⚠{"reqSignature":{},"txn":{"data":{"data":{"alias":"IdRamp-Main","blskey":"4S14ydD6SJzfmBHBDJNKGmdzPDa5kh2mAQLmVGRCLXkQWyQErSfmS1aYDzYA25SAqg2cbfsQku1ZvwqXDRcsbEsdkNRHM66VhdceFv9B4aPQHBQsDkB2536bd5XKgvnR41AgN5Ck9AM43y5hiyWBWy8PSc22wNtNq4JuQs4f3JMcvAp","blskey_pop":"RAYwNv1BMhf7qAsprPiqo5gEsAWSRgYH8u3kLYr564UprkvZLMGuAySzDr3KAAViuWY76X3HHPTY6wWVT8ZehoJMBTSHknMqgNAiivH32zHFfFuoFdfJLVWKKJ4zs4F4RZ9nqrfwm62bBnqcYoGbepNxmYamNZW2SEwiyuGRBNFjHy","client_ip":"207.126.128.13","client_port":"9702","node_ip":"207.126.129.13","node_port":"9701","services":["VALIDATOR"]},"dest":"35g3d2MxTmWfjpMEBJA2RWBzdViBa7xsiC8buHWeMDDt"},"metadata":{"from":"Ge6GrQRLzGEAjtSBeGL9ku"},"type":"0"},"txnMetadata":{"seqNo":2,"txnId":"d0a1a8622e3200da62d0fd3a2dd8676e131164244d8e381c9968bbe17e514ccf"},"ver":"1"}
```
Comment (JamesKEbert) [inline core/configs/ledgers/indy/indicio-main-net/genesis-file.ts]: Could you fix this typo @seajensen

**Review by JamesKEbert — CHANGES_REQUESTED** [review]
LGTM besides a minor typo

**Review by JamesKEbert — APPROVED** [review]
👍

**Pull request merged**